state/forkid.go

```diff
@@ -17,6 +23,37 @@ func (s *State) UpdateForkIDIntervals(intervals []ForkIDInterval) {
 	s.cfg.ForkIDIntervals = intervals
 }
 
+// AddForkIDInterval updates the forkID intervals
+func (s *State) AddForkIDInterval(ctx context.Context, newForkID ForkIDInterval, dbTx pgx.Tx) error {
+	// Add forkId to db and memori variable
+	oldForkIDs, err := s.GetForkIDs(ctx, dbTx)
+	if err != nil {
+		log.Error("error getting oldForkIDs. Error: ", err)
+		return err
+	}
+	if len(oldForkIDs) == 0 {
+		s.UpdateForkIDIntervals([]ForkIDInterval{newForkID})
+	} else {
+		var forkIDs []ForkIDInterval
```

Comment:
Could be is not necessary, but we need to check that newForkID.ForkId is just the next one (+1) of oldForkIDs[len(oldForkIDs)-1].ForkId, if not we can return an error. With this we ensure we are adding the next forkid in the list

Reply:
Done